tests/unit/ConsentContinuationOut.spec.ts

```diff
@@ -8,17 +8,57 @@ import { ConfirmDialog, PaymentErrorDialog, ResumeErrorDialog, SaveErrorDialog,
 import { Certify, DetailComment } from '@/components/common'
 import { CourtOrderPoa } from '@bcrs-shared-components/court-order-poa'
 import { DocumentDelivery } from '@bcrs-shared-components/document-delivery'
+import axios from '@/axios-auth'
+import flushPromises from 'flush-promises'
 import mockRouter from './mockRouter'
+import sinon from 'sinon'
 import VueRouter from 'vue-router'
 
+// suppress various warnings:
+// - "Unknown custom element <affix>" warnings
+// - "$listeners is readonly"
+// - "Avoid mutating a prop directly"
+// ref: https://github.com/vuejs/vue-test-utils/issues/532
+Vue.config.silent = true
+
 Vue.use(Vuetify)
 
 const store = getVuexStore() as any // remove typings for unit tests
 const localVue = createLocalVue()
 localVue.use(VueRouter)
 
 describe('Consent to Continuation Out view', () => {
+  let sinonAxiosGet: any
+
   beforeEach(() => {
+    sinonAxiosGet = sinon.stub(axios, 'get')
+
+    store.commit('setTestConfiguration', { key: 'VUE_APP_PAY_API_URL', value: 'https://pay.web.url/' })
+
+    // mock "get orig filing" endpoint
+    sinonAxiosGet
+      .withArgs('businesses/CP1234567/filings/123')
+      .returns(new Promise(resolve =>
```

Comment:
This is the "old way" of mocking a get call.

The new way is to just mock the LegalServices call. After all, you don't need to test Axios (and it's cleaner, too). Example: https://github.com/bcgov/business-filings-ui/blob/0cae8df3d15f5f5c6510d6381d611ee3b006ed39/tests/unit/BusinessActions.spec.ts#L20

Reply:
Alright, noted! I looked also at various other examples in the project. I'll be keeping this in mind.

Reply:
This is the oldest UI so you'll find some older ways of doing things 😁

Reply:
I managed to update my unit test to use the new way. I like it 🕺

Reply:
Beauty.